Shared/Tests.Shared/RelationshipTests.cs

```diff
@@ -479,6 +479,23 @@ public void Insert_WhenIndexIsEqualToCount_ShouldWork()
             Assert.That(person.Dogs.Count, Is.EqualTo(2));
         }
 
+        [Test]
+        public void Backlinks()
```

Comment:
It seems we're testing a backlink to a List, should we add a test for a `RealmObject`?